dist/access_check/check_sources.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
Suggest adding a description/docstring for the script to help our future selves.